src/detection/os/os_linux.c

```diff
@@ -89,6 +89,21 @@ FF_MAYBE_UNUSED static void getUbuntuFlavour(FFOSResult* result)
         ffStrbufSetStatic(&result->idLike, "ubuntu");
     }
 
+    if (ffPathExists("/etc/lliurex-cdd-version", FF_PATHTYPE_FILE))
+	{
+        ffStrbufSetStatic(&result->name, "LliureX");
+        ffStrbufSetStatic(&result->id, "lliurex");
+        ffStrbufClear(&result->version);
+        if (ffProcessAppendStdOut(&result->version, (char* const[]) {
+            "/usr/bin/lliurex-version",
+            NULL,
+        }) == NULL) // 8.2.2
+            ffStrbufTrimRightSpace(&result->versionID);
```

Comment:
You appended the version string to `&result->version`, but triming `&result->versionID` here?

Reply:
And you put executable binaries to `/etc/`? Shouldn't it be a regular text file?